minetest/minetest
# Properly keep authority over attached player position server-side

See #10908

## To do

This PR is a Work in Progress

## How to test

1. Create a server
2. Check whether you can see other players moving
3. Place cart onto rail, attach
4. [this script](https://github.com/minetest/minetest/issues/10908#issuecomment-774481432)

On low latency connections there are now less anticheat false-positives, but I'm afraid the problem will persist with higher ping. Suggestions to improve this are welcome.
